### PR TITLE
[Documentation] Contributing: use RFC process for new features

### PR DESCRIPTION
**Pull request by pixelhandler**
The Feature Request section of the contributing guide needed links and instruction for the Ember RFC process.

[requesting-a-feature section, rendered](https://github.com/pixelhandler/ember.js/blob/contributing-feature-requests/CONTRIBUTING.md#requesting-a-feature)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -47,22 +47,25 @@ there is a bug and the faster we'll be able to take action.
 
 ## Requesting a Feature
 
-1. Search Issues for similar feature requests. It's possible somebody has
```
Comment (stefanpenner) [inline CONTRIBUTING.md]: one adjustment, would be:

1. RFC PR for well formed ideas
2. RFC issues for proposed rough ideas – basically a great place to test the waters.

Comment (pixelhandler) [inline CONTRIBUTING.md]: @stefanpenner I added those two bullet points.

Comment (stefanpenner) [inline CONTRIBUTING.md]: awesome

**Comment by stefanpenner** [comment]
one last thing, it looks like the original document attempted to abide by 80char lines, if you could re-align that we can merge.

**Comment by pixelhandler** [comment]
@stefanpenner I cleaned up the lines to use 80 chars

**Comment by stefanpenner** [comment]
re-running the CI. no idea why it failed :P

**Comment by stefanpenner** [comment]
@trek mind doing a once-over ?

**Comment by pixelhandler** [comment]
@trek what say you my friend?

**Comment by trek** [comment]
LGTM.

**Comment by trek** [comment]
It's amazing we how long we've had RFCs without mentioning them here!

**Comment by pixelhandler** [comment]
@trek @stefanpenner thanks :) have a great weekend.

**Pull request merged**